src/component/parser/json/converter/utils.ts

```diff
@@ -124,21 +122,29 @@ export class ConvertedElements {
   }
 
   private categoryValues: Map<string, CategoryValueData> = new Map();
-  registerCategoryValues(id: string, value: string): void {
-    this.categoryValues.set(id, { value });
+  findCategoryValue(categoryValue: string): CategoryValueData {
+    return this.categoryValues.get(categoryValue);
   }
-
-  // Special case: create the ShapeBpmnElement instance here to avoid duplication in CollaborationConverter and ProcessConverter
-  buildShapeBpmnGroup(groupBpmnElement: TGroup, processId?: string): ShapeBpmnElement | undefined {
-    const categoryValueData = this.categoryValues.get(groupBpmnElement.categoryValueRef);
-    if (categoryValueData) {
-      return new ShapeBpmnElement(groupBpmnElement.id, categoryValueData.value, ShapeBpmnElementKind.GROUP, processId);
-    }
-    this.parsingMessageCollector.warning(new GroupUnknownCategoryValueWarning(groupBpmnElement.id, groupBpmnElement.categoryValueRef));
-    return undefined;
+  registerCategoryValue(id: string, value: string): void {
+    this.categoryValues.set(id, { value });
   }
 }
 
+// Special case: create the ShapeBpmnElement instance here to avoid duplication in CollaborationConverter and ProcessConverter
```

Comment:
ℹ️ I think we can remove the comment as the function is now out of the `ConvertedElements` class.
In `ConvertedElements` we don't want to create instance  of model elements, so we added the comment to mention we didn't follow the rule on purpose.